services/src/handlers/ebv.rs

```diff
@@ -1390,7 +1387,7 @@ mod tests {
         assert_eq!(res.status(), 200, "{:?}", res.response());
 
         assert_eq!(
-            read_body_string(res).await,
+            serde_json::from_str::<Value>(&read_body_string(res).await).unwrap(),
```

Comment:
Hm, you could create a `read_body_json` method.

You do this a lot of times, maybe search and replace, then.

Reply:
589ee064e685c5ae0df242f6075009be75e5b421